config.json

```diff
@@ -218,20 +218,21 @@
         ]
       },
       {
-        "slug": "allergies",
-        "name": "Allergies",
-        "uuid": "52b1856c-7570-4d75-aaa9-e1cff3cb6155",
-        "difficulty": 3,
+        "slug": "triangle",
+        "name": "Triangle",
+        "uuid": "58e138e9-55d8-49fa-b05d-70a6aa627b21",
+        "difficulty": 1,
         "practices": [],
         "prerequisites": [],
         "topics" : [
-          "logicals",
+          "bools",
           "conditionals",
           "comparisons",
           "numbers",
-          "bitwise_operations"
+          "sequences",
+          "interfaces"
         ]
-      } 
+      }      
```

Comment:
Why the additional whitespace here?

Reply:
I actually do not know where that whitespace came from. Removed it.